pkg/plugin/processor/builtin/impl/webhook/http.go

```diff
@@ -44,8 +44,10 @@ type httpConfig struct {
 	URL string `json:"request.url" validate:"required"`
 	// Method is the HTTP request method to be used.
 	Method string `json:"request.method" default:"GET"`
-	// The value of the `Content-Type` header.
+	// Deprecated: use `headers.Content-Type` instead.
 	ContentType string `json:"request.contentType" default:"application/json"`
+	// Headers to add to the request, use headers.* to specify the key and set its value, ex: "headers.Authorization"="Bearer key".
+	Headers map[string]string
```

Comment:
I'd specify the `json` tag explicitly, just to be sure we don't rely on the implicit behavior.

```suggestion
	Headers map[string]string `json:"headers"`
```